__main__.py

```diff
@@ -27,15 +33,88 @@ async def echo_message(client: Client, message: Message) -> None:
     )
 
 
+async def add_or_remove(client: Client, message: Message) -> None:
+
+    if not await is_chat(message.text):
+        return "No such chat-group were found"
```

Comment:
Видимо функция возращает строку:
```suggestion
async def add_or_remove(client: Client, message: Message) -> str:

    if not await is_chat(message.text):
        return "No such chat-group were found"
```